src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx

```diff
@@ -64,6 +65,7 @@ export const MetaNavigation = connect(function MetaNavigation({
           <CurrentUserDropdown widget={widget} root={root} />
         </Nav>
       )}
+      <SearchBox searchResultsPage={root.get('siteSearchResultsPage')} />
```

Comment:
You're now rendering the `SearchBox` twice and hiding one of them according to the current viewport? Is there maybe another solution to this?

Reply:
Sure.. we can introduce a real grid for whole navbar section and define grid areas. Everything custom - not part of Bootstrap.